ingestion/src/metadata/utils/column_type_parser.py

```diff
@@ -145,6 +145,10 @@ def get_column_type(column_type: Any) -> str:
         type_class: Optional[str] = None
         if isinstance(column_type, types.NullType):
             return "NULL"
+        for sql_type in ColumnTypeParser._COLUMN_TYPE_MAPPING.keys():
+            if isinstance(column_type, sql_type):
+                type_class = ColumnTypeParser._COLUMN_TYPE_MAPPING[sql_type]
```

Comment:
Not all types are the same to achieve what you intended.

![스크린샷 2022-02-04 오전 1 32 38](https://user-images.githubusercontent.com/10463931/152385844-9ff6f972-7cfb-4280-99af-5365f685d392.png)
`

Reply:
Hey @joswlv, we parse the raw data type from data sources, it would be helpful to know which hive type it is that we are not covering. Can you tell us which type wasn't getting parsed?

Reply:
@ayush-shah 
what i want to say is The `if isinstance(column_type, sql_type):` syntax doesn't work.
It seems that you need to use a different comparison method.
![스크린샷 2022-02-04 오전 10 01 04](https://user-images.githubusercontent.com/10463931/152454951-9b83c91c-543b-4586-bb85-154c5da745ef.png)

Reply:
@pmbrull any suggestions here

Reply:
Sorry for the late response. Maybe we could have an enum wrapping the types instead of a Dict:

```python
from sqlalchemy.sql import sqltypes
from enum import Enum

class TypeMapping(Enum):
    ARRAY = sqltypes.ARRAY
    BOOLEAN = sqltypes.Boolean
    
    def equals(self, _type):
        return self.value == _type

TypeMapping.ARRAY.equals(sqltypes.ARRAY)
# >>> True
```

Reply:
Then:
```python
from typing import Any

def find_type(column_type: Any):
    for _type in TypeMapping:
        if _type.equals(column_type):
            return _type.value
    
    raise ValueError(f"Cannot find type {column_type}")

print(find_type(sqltypes.ARRAY))
# >>> <class 'sqlalchemy.sql.sqltypes.ARRAY'>
print(find_type(sqltypes.CHAR))
# >>> ValueError
```